config/crd/bases/postgresql.cnpg.io_clusters.yaml

```diff
@@ -2104,6 +2104,12 @@ spec:
                   - name
                   type: object
                 type: array
+              failoverDelay:
+                default: 0
+                description: The time in seconds a primary PostgreSQL instance should
+                  be deemed unhealthy before triggering a failover
```

Comment:
I would rephrase this a little bit. Maybe something like: _The amount of time (in seconds) to wait before triggering a failover after the primary PostgreSQL instance in the cluster was detected to be unhealthy_